src/stan/lang/grammars/statement_grammar_def.hpp

```diff
@@ -192,12 +204,47 @@ namespace stan {
       for_statement_r
```

Comment:
Thanks, @bob-carpenter ! Is this the sort of use of a local variable you mean? My intention is to store the identifier in `_a`. I'll look into the difference between `>` and `>>`. Thanks for explaining. Could we maybe plan a time on Monday or Tuesday (or Wednesday before 3:30pm) to sit down together and go through some compiler code? It'd be nice to get to the point where I understand it well enough myself to be productive independently. Wednesday will be my last day in NYC for this trip sadly. Also, do you still want to try to meet Andrew together?